src/lava/proc/conv/process.py

```diff
@@ -30,6 +30,8 @@ class Conv(AbstractProcess):
         convolution dilation. Default is 1.
     groups : int
         number of groups in convolution. Default is 1.
+    graded_spike: bool
```

Comment:
Perhaps rename `graded_spike` to something more verbose like `enable_graded_spikes` or `graded_spikes_enabled`. At the first glance, it reads like graded spikes are bools, which is obviously wrong.

Reply:
That's a good point. Changed them to use_graded_spike